Electrometer/v4/tts_cimacs1_v3.yaml

```diff
@@ -0,0 +1,5 @@
+# Testing of electrometer with new converter
```

Comment:
filename should be v4.
Also, do we need to augment the version? are these not backwards compatible?

Reply:
I would recommend to drop the version from the filename since it is implicit from the directory.

Reply:
I agree with Wouter. Having the schema version in the file name is redundant with the directory referring to the schema version. I also assert that the schema version is not necessarily about backwards compatibility but about a revision of the schema. Unless you can point me to a document that does define what a schema version is.

Reply:
I think dropping the version suffix is ok.
@tribeiro - is there a defined procedure on when schema versions get augmented? Whatever the answer is we should write it down somewhere!